core/admin.py

```diff
@@ -12,9 +14,25 @@ class EventAdmin(admin.ModelAdmin):
 
     def get_description_as_text(self, obj):
         return mark_safe(obj.description)
-    
-    get_description_as_text.short_description = 'Description'
+
+    get_description_as_text.short_description = "Description"
+
 
 admin.site.register(Event, EventAdmin)
 
 admin.site.register(Venue)
+
+
+class StaticPageAdmin(admin.ModelAdmin):
+    list_display = ("url_path", "title", "get_content_as_text")
+    search_fields = ("url_path", "title", "content")
+
+    def get_content_as_text(self, obj):
+        # extract the first 100 characters, making
+        # sure to skip tags
+        return re.sub(r"<[^>]*>", "", obj.content)[:100] + "..."
+
```

Comment:
When I create some static pages, the descriptions in the admin list sometimes still contain &nbsp; instead of spaces. Not the biggest issue, but could be worth looking at.

Reply:
sorry, did you mean `&nbsp;`? here on github, the `&nbsp;` is appearing as a ... "nbsp" and it's actually not visible :-)

do you mean that you see nbsp instead of spaces in the markup code? are you using the tiny mce plugin in the admin to create the static page?